src/6_branch.js

```diff
@@ -693,7 +696,7 @@ Branch.prototype['logout'] = wrap(callback_params.CALLBACK_ERR, function(done) {
 		self.identity = data['identity'];
 		session.update(self._storage, data);
 
-		done(err);
+		done(null);
```

Comment:
Quick note on this, there is an earlier check for err that would call `done(err)`. Since err will be undefined or null at this point, I'm forcing it to null for consistency